sdcm/nemesis.py

```diff
@@ -4233,6 +4233,7 @@ def disrupt(self):
 class ToggleGcModeMonkey(Nemesis):
     kubernetes = True
     disruptive = False
+    run_with_gemini = False
```

Comment:
This solution is workarounding the issue described in task:
 The new test might execute the nemesis of ToggleGcModeMonkey the will eliminate the whole purpose of this test.

Reply:
does it meant that, it will run only when Gemini is false?

Reply:
It means that it will run in all tests that does NOT have `nemesis_selector: ['run_with_gemini']`

Reply:
shouldn't that be the opposite? and run only when we have `nemesis_selector: ['run_with_gemini']` ?

Reply:
i'm lost here :cry:

Reply:
@yarongilor is this nemesis is going to be run at any longevity ? i.e. what you changed here it mean it would never run on any gemini test, not just the once you created.

Reply:
@fruch , exactly. That is one 'ugly' workaround to solve the issue not addressed by @roydahan 's suggestion.
It might not be such a big deal, and perhaps even no impact at all.

Reply:
again, you want this one to run on all test cases ? and not on this test ?

so there are those options:

1) add a flag for those, so it can be filter where needed

2) add a filter by Class name or method name, that work across all nemesis

3) remove this nemesis completely, and let this be tests only on those specific test you introduce.

Any I'll let @roydahan comment on the PR, is it's follows what you agreed upon.

Reply:
According to Roy `changing run_with_gemini = False to ToggleGcModeMonkey isn't ugly solution, it's fine for now.`

Reply:
Please refer me to this discussion, I need to refresh my memory...